src/module.cc

```diff
@@ -212,7 +212,11 @@ static void PrintStackFromStackTrace(Isolate* isolate, FILE* fp) {
                                                           StackTrace::kDetailed);
   // Print the JavaScript function name and source information for each frame
   for (int i = 0; i < stack->GetFrameCount(); i++) {
-    Local<StackFrame> frame = stack->GetFrame(i);
+    Local<StackFrame> frame = stack->GetFrame(
+#if defined(V8_MAJOR_VERSION) && (V8_MAJOR_VERSION >= 7)
```

Comment:
There are versions of V8 that don't define V8_MAJOR_VERSION?

Reply:
Probably not in any versions of Node.js this module has ever supported. It may be true in long outdated versions of Node.js (e.g. `deps/v8/include/v8-version.h` (where `V8_MAJOR_VERSION` is currently defined) doesn't exist in Node.js' `v0.10` branch).

For reference, `nan` does similar checks: https://github.com/nodejs/nan/blob/19f08c2394b19a5dbe854020fed0446606d68955/nan.h#L215-L216